### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/Elhebert/laravel-sri/pull/78/commits/30ea2dab07c7cc3f90f3a25239b4b9487e0ab0e9?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-sri.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "elhebert/laravel-sri": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/phpunit-l8.yml [hunk 1]
```diff
@@ -16,11 +16,13 @@ jobs:
             fail-fast: false
             matrix:
                 php: [8.0, 7.4, 7.3]
-                laravel: [8.*]
+                laravel: [9.*, 8.*]
```
Comment (jasonmccreary) [inline .github/workflows/phpunit-l8.yml:19]: If you plan to drop Laravel 8 support you may streamline this build matrix to just PHP 8, Laravel 9, and Testbench 7.

File: composer.json [hunk 2]
```diff
@@ -14,12 +14,12 @@
     ],
     "require": {
         "php": "^7.3 | ^7.4 | ^8.0",
```
Comment (jasonmccreary) [inline composer.json:16]: Ditto here for PHP 8.

**Comment by Elhebert** [comment]
Thank you @jasonmccreary !

**Pull request merged**